modules/search/instant-search/components/search-results.jsx

```diff
@@ -9,17 +9,50 @@ import { h, Component } from 'preact';
 /**
  * Internal dependencies
  */
-import SearchResult from './search-result';
+import SearchResultMinimal from './search-result-minimal';
 
 class SearchResults extends Component {
+	render_result( result ) {
+		switch ( this.props.resultFormat ) {
+			case 'engagement':
+			case 'product':
+			case 'minimal':
+			default:
+				return <SearchResultMinimal result={ result } />;
+		}
+	}
+
 	render() {
-		const { results = [], query } = this.props;
+		const { results = [], query, total = 0, corrected_query = false } = this.props;
+		if ( query === '' ) {
```

Comment:
Shouldn't we show "no results", or not output anything if the search keyword is empty?